tests/suites/helpers.function

```diff
@@ -111,7 +111,7 @@ typedef UINT32 uint32_t;
 
 static struct
 {
-    int failed;
+    int result; /* 0 on success, 1 on failure, 2 on skip */
```

Comment:
@hanno-arm isn't the idea to have an ```enum``` instead of ```int``` to enforce only allowed statuses.

Reply:
Please change `result` to an enum.

Reply:
Note: I have done this change in https://github.com/ARMmbed/mbedtls/pull/2124